### ISSUE TITLE
NPE on opening multi properties file after plugin activation

### ISSUE THREAD

**Issue by jkovalsky** [lsimedia/MultiProperties#3]
**Description:**
When opening non-empty multi properties file the IDE throws java.lang.NullPointerException if the plugin was reactivated.

**Environment:**
Product Version: Apache NetBeans IDE 11.0 (Build incubator-netbeans-release-404-on-20190319)
Updates: Updates available
Java: 1.8.0_201; Java HotSpot(TM) 64-Bit Server VM 25.201-b09
Runtime: Java(TM) SE Runtime Environment 1.8.0_201-b09
System: Linux version 4.15.0-52-generic running on amd64; UTF-8; cs_CZ (nb)

**Steps to reproduce:**

1. Launch Apache NetBeans 11.0, activate JavaSE cluster, and install nb-javac plugin.
2. Install http://plugins.netbeans.org/plugin/63739 into the IDE via _Tools > Plugins > Downloaded > Add Plugins... > Install_
3. Create Maven based Java application, save attached _dictionary.multiproperties_ file in its _Source Packages_ and make sure it opens nicely in Editor.
4. Via Tools > Plugins > Installed > Deactivate the MultiProperties plugin.
5. Make sure that the _dictionary.multiproperties_ file opens as a plain XML file only in Editor.
6. Via Tools > Plugins > Installed > Activate the MultiProperties plugin again.
7. Try to open the _dictionary.multiproperties_ file in Editor.

**Expected:** The file opens smoothly as in step 3.
**Actual:** NPE is thrown and the file remains opened only as a plain XML file in Editor.

**Comment by jkovalsky** [comment]
[NullPointerException.txt](https://github.com/lsimediasarl/MultiProperties/files/3311760/NullPointerException.txt)

**Comment by jkovalsky** [comment]
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<MultiProperties xmlns="hu.skzs.multiproperties">
    <Version/>
    <Name/>
    <Description/>
    <Handler>Empty</Handler>
    <Columns>
        <Key>
            <Width>75</Width>
        </Key>
        <Column>
            <Width>75</Width>
            <Name>en</Name>
            <Description/>
            <HandlerConfiguration/>
        </Column>
        <Column>
            <Width>75</Width>
            <Name>cz</Name>
            <Description/>
            <HandlerConfiguration/>
        </Column>
    </Columns>
    <Records>
        <Property>
            <Name>info.greeting</Name>
            <Description/>
            <Disabled>false</Disabled>
            <MultiLine>true</MultiLine>
            <DefaultValue>Ahoj</DefaultValue>
            <Value disabled="false" final="false">Hello</Value>
            <Value disabled="false" final="false">Ahoj</Value>
        </Property>
    </Records>
</MultiProperties>

```

**Comment by sbodmer** [comment]
Thanks for the detailed bug report.
I found the problem, the Version node is empty and I did not check it's content.
A temporary fix is to set the node `<Version>1.2</Version>.`
I will deploy a fix shortly (1.0.0.5).
